src/debug.ts

```diff
@@ -44,7 +44,7 @@ export class PreLaunchTaskProvider implements vscode.TaskProvider
     {
         let ws = context.workspaces.folderToWorkspace.get(pkg.path)
         if (!ws) {
-            throw new Error("package not in a workspace");
+            return;
```

Comment:
Why not an error? Haven't we discusses that packages should be in a workspace ?

My problem here is "button silently doing nothing".

Reply:
This method is called whenever the task picker ui is shown or any task is run (by provideTasks) so, you get an error just by having a package that is not in an autoproj workspace (even if you are not doing anything to it). Moreover, it won't make the button "silently do nothing", the debugging session will still be launched..

We probably should throw an error and abort the debugging session but the place to it is [here](https://github.com/rock-core/vscode-rock/blob/master/src/packages.ts#L520) otherwise we are throwing out-of-context errors..

Reply:
Actually, this is only valid for orogen packages and those are guaranteed to be in a workspace by the PackageFactory...